packages/jest-cli/src/lib/init/questions.js

```diff
@@ -8,7 +8,7 @@
  */
 
 type Question = {
-  intial?: boolean | number,
+  initial?: boolean | number,
```

Comment:
Thanks! This exposes that we didn't type this correctly 😅 I'll quickly fix it

Reply:
LOL! Thanks... you just got to approve my PR and merge ... :)

Reply:
@SimenB thanks for approving :)